GoogleCloudPlatform/java-docs-samples
# Misc testing fixes.

* Improve readability of run_tests.sh
* Removed diff_tests since no longer used
* Removed parent poms since no longer used. 
* Update shared config version of everything to 1.0.11
* Update several plugins causing conflicts
